src/unity/lib/visualization/summary_view.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef _TC_SUMMARY_VIEW
+#define _TC_SUMMARY_VIEW
+
+#include <flexible_type/flexible_type.hpp>
+#include <parallel/lambda_omp.hpp>
+
+#include "transformation.hpp"
+
+namespace turi {
+namespace visualization {
+
+class summary_view_transformation_output : public transformation_output {
+  private:
+    std::vector<std::shared_ptr<transformation_output>> m_outputs;
+  public:
+    std::vector<std::string> column_names;
```

Comment:
Please prefix class member names with `m_`. It's also generally not good practice to reveal data members directly from classes like that, but this code is fairly internal so it's fine for now.

Reply:
prefixed